chapters/version_control_with_git/chapter.md

```diff
@@ -160,17 +160,17 @@ Initialised empty Git repository in /home/bilderbuchi/demoProject/.git/
 
 #### `.gitignore`
 
-One thing we should do right at the beginning is add a special Git file called [`.gitignore`](http://git-scm.com/docs/gitignore) to the root of our repository.
+One thing we should do right at the beginning adds a special Git file called [`.gitignore`](http://git-scm.com/docs/gitignore) to the root of our repository.
 
 It's important that the Git repository contains all files necessary to successfully compile our program, but no unnecessary stuff.
-Generally, this means that files we edit by hand (e.g. source and header files, Readme files, images,...) should be included in the repository, but files which are *generated from* our code (e.g. compiled binaries, pdfs generated from some source file, video files or image sequences created with our program) should stay out.
+Generally, this means that files we edit by hand (e.g. source and header files, Readme files, images,...) should be included in the repository, but files which are *generated from* our code (e.g. compiled binaries, pdf generated from some source file, video files or image sequences created with our program) should stay out.
```

Comment:
Multiple pdf files may be generated from one source file, so I lean slightly towards the plural form, but only slightly.